### PR TITLE
Adds hello and bye to readme

### PR DESCRIPTION
**Pull request by haroun-b**

### CODE REVIEW

File: README.md
Comment (clairelacanal) [inline README.md:1]: c'EST QUOI ÇA !?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+hello
```
Comment (clairelacanal) [inline README.md:1]: COUCOU :)

**Review by 1yasmou — COMMENTED** [review]
ok

File: README.md
Comment (haroun-b) [reply README.md:1]: C'est du code

**Review by 1yasmou — APPROVED** [review]

**Review by clairelacanal — APPROVED** [review]

**Pull request merged**